src/views/InvolvementProfile/InvolvementProfile.module.scss

```
@@ -11,3 +11,26 @@
 .update_image_components {
   margin-top: 15px;
 }
+
+_removeImage {
```

You need to title CSS classes starting with a dot, so these probably aren't working right now (did you test it to make sure it looked right?)  You should also probably get rid of the underscores here.  They aren't needed anymore since we aren't using the nested styles. 
```suggestion
.removeImage {
```